apps/bitcoin/common/src/psbt.rs

```diff
@@ -478,15 +478,24 @@ pub fn fill_psbt_with_bip388_coordinates(
 
 #[cfg(test)]
 mod tests {
-    use core::str::FromStr;
+    use base64::{engine::general_purpose::STANDARD, Engine as _};
 
     use super::*;
 
     const TEST_PSBT: &'static str = "cHNidP8BAFUCAAAAAVEiws3mgj5VdUF1uSycV6Co4ayDw44Xh/06H/M0jpUTAQAAAAD9////AXhBDwAAAAAAGXapFBPX1YFmlGw+wCKTQGbYwNER0btBiKwaBB0AAAEA+QIAAAAAAQHsIw5TCVJWBSokKCcO7ASYlEsQ9vHFePQxwj0AmLSuWgEAAAAXFgAUKBU5gg4t6XOuQbpgBLQxySHE2G3+////AnJydQAAAAAAF6kUyLkGrymMcOYDoow+/C+uGearKA+HQEIPAAAAAAAZdqkUy65bUM+Tnm9TG4prer14j+FLApeIrAJHMEQCIDfstCSDYar9T4wR5wXw+npfvc1ZUXL81WQ/OxG+/11AAiACDG0yb2w31jzsra9OszX67ffETgX17x0raBQLAjvRPQEhA9rIL8Cs/Pw2NI1KSKRvAc6nfyuezj+MO0yZ0LCy+ZXShPIcACIGAu6GCCB+IQKEJvaedkR9fj1eB3BJ9eaDwxNsIxR2KkcYGPWswv0sAACAAQAAgAAAAIAAAAAAAAAAAAAA";
 
+    fn psbt_from_str(psbt: &str) -> Result<Psbt, String> {
+        let decoded = STANDARD
+            .decode(psbt)
+            .map_err(|e| format!("Failed to decode PSBT: {}", e))?;
+        let psbt = Psbt::deserialize(&decoded)
+            .map_err(|e| format!("Failed to deserialize PSBT: {}", e))?;
+        Ok(psbt)
+    }
+
     #[test]
     fn test_set_and_get_account_name() {
-        let mut psbt = Psbt::from_str(TEST_PSBT).unwrap();
```

Comment:
If all the tests use the same PSBT, why not just `fn test_psbt() -> Psbt`?

(FTR, `TEST_PSBT.parse::<Psbt>()` would've been more idiomatic in previous code.)

Reply:
The last test has a different PSBT, actually.